integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -31,11 +31,12 @@ def test_multiple_granules_happy_path(self):
             cumulus_bucket_name = "orca-sandbox-s3-provider"
             recovery_bucket_name = helpers.recovery_bucket_name
             excluded_filetype = []
+
             name_1 = "ancillary_data_input_forcing_ECCO_V4r4.tar.gz"
             key_name_1 = "PODAAC/SWOT/" + name_1
             file_1_hash = uuid.uuid4().__str__()
             file_1_hash_type = uuid.uuid4().__str__()
-            name_2 = "MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"
+            name_2 = "MOD09GQ.A2017025.h21v00.006.2017034065104_ndvi.jpg"
```

Comment:
Two tests were using this file, causing a race condition. Works fine with the new name.

Reply:
file randomization should fix this issue